include/openrave/kinbody.h

```diff
@@ -854,15 +854,20 @@ class OPENRAVE_API KinBody : public InterfaceBase
             bool operator ==(const DOFFormat& r) const;
             boost::shared_ptr<Joint> GetJoint(KinBody &parent) const;
             boost::shared_ptr<Joint const> GetJoint(const KinBody &parent) const;
+            std::string to_string() const;
```

Comment:
@tgn3000 what's the difference between to_string and getting the JSON struct and dumping it? You want to display shorter information? If so, what's the deciding factor for this shorter info?
Can we also give the function a nicer name following mujin conventions?
thanks

Reply:
Reduced log and removed `to_string()` related stuff. Tried JSON thing; did not work.

This is `kinbody.h`, so cannot `#include <openrave/openravejson.h>` to inherit `JsonSerializable`. Without inheriting, I cannot use `SaveJsonValue` unless applying the SFINAE technique. Tried and it worked, but slightly fancy.